<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>486</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>810</width>
    <height>486</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="whole_Frame">
      <property name="minimumSize">
       <size>
        <width>792</width>
        <height>444</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>792</width>
        <height>444</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="whole_group">
         <item>
          <layout class="QHBoxLayout" name="Top_group">
           <item>
            <widget class="QPushButton" name="File_Select">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>70</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>70</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>한컴 고딕</family>
               <pointsize>10</pointsize>
               <weight>9</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">border-style: solid;
background-color: rgb(148, 214, 255);
border-width: 2px;
font: 75 10pt &quot;한컴 고딕&quot;;
border-color: #FA8072;
border-radius: 3px</string>
             </property>
             <property name="text">
              <string>파일 선택</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="current_File">
             <property name="font">
              <font>
               <family>한컴 고딕</family>
               <pointsize>12</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">border-style: solid;
background-color: rgb(255, 255, 255);
border-width: 2px;
border-color: #FA8072;
border-radius: 3px</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="text">
              <string>현재 파일:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="middle_group_2">
           <item>
            <widget class="QFrame" name="middle_Frame">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>200</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Plain</enum>
             </property>
             <property name="lineWidth">
              <number>1</number>
             </property>
             <widget class="QPushButton" name="select_save_place">
              <property name="geometry">
               <rect>
                <x>330</x>
                <y>80</y>
                <width>120</width>
                <height>50</height>
               </rect>
              </property>
              <property name="maximumSize">
               <size>
                <width>120</width>
                <height>50</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">border-style: solid;
background-color: rgb(148, 214, 255);
border-width: 2px;
font: 75 10pt &quot;한컴 고딕&quot;;
border-color: #FA8072;
border-radius: 3px</string>
              </property>
              <property name="text">
               <string>파일 저장 경로 지정</string>
              </property>
             </widget>
             <widget class="QTextBrowser" name="show_save_place_2">
              <property name="geometry">
               <rect>
                <x>470</x>
                <y>0</y>
                <width>301</width>
                <height>200</height>
               </rect>
              </property>
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>155</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>500</width>
                <height>200</height>
               </size>
              </property>
              <property name="html">
               <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Gulim'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;저장경로:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
             <widget class="QFrame" name="middle_2Frame">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>311</width>
                <height>201</height>
               </rect>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Plain</enum>
              </property>
              <widget class="QWidget" name="layoutWidget_3">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>311</width>
                 <height>201</height>
                </rect>
               </property>
               <layout class="QVBoxLayout" name="choose_listofFile">
                <property name="sizeConstraint">
                 <enum>QLayout::SetDefaultConstraint</enum>
                </property>
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <item alignment="Qt::AlignHCenter">
                 <widget class="QLabel" name="label_2">
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <family>한컴 고딕</family>
                    <pointsize>12</pointsize>
                    <weight>9</weight>
                    <italic>false</italic>
                    <bold>false</bold>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color: blue;
font: 75 12pt &quot;한컴 고딕&quot;;</string>
                  </property>
                  <property name="text">
                   <string>변환하고자 하는 파일 형식을 선택하시오.</string>
                  </property>
                 </widget>
                </item>
                <item alignment="Qt::AlignHCenter">
                 <widget class="QComboBox" name="Select_File_type_2">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>200</width>
                    <height>50</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>200</width>
                    <height>50</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">border-style: solid;
border-width: 2px;
border-radius: 3px
</string>
                  </property>
                  <item>
                   <property name="text">
                    <string>hwp</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>txt</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>img</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>word</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>ppt</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="bottom_group">
           <item>
            <widget class="QPushButton" name="change_File">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>70</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>70</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>한컴 고딕</family>
               <pointsize>12</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">border-style: solid;
background-color: rgb(148, 214, 255);
border-width: 2px;
border-color: #FA8072;
border-radius: 3px</string>
             </property>
             <property name="text">
              <string>변환</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="show_result">
             <property name="font">
              <font>
               <family>한컴 고딕</family>
               <pointsize>12</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">border-style: solid;
border-width: 2px;
border-color: #FA8072;
border-radius: 3px</string>
             </property>
             <property name="text">
              <string>변환 결과:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
